docs/release-notes/python-39-bump.rst

```diff
@@ -0,0 +1,5 @@
+:orphan:
+
+**Improvements**
+
+-  Our default environment images have been updated to Python 3.9. (Previously Python 3.8)
```

Comment:
:orphan:

**Improvements**

-  Update default environment images to Python 3.9 from Python 3.8.